outport/process/transactionsfee/transactionsFeeProcessor_test.go

```diff
@@ -597,3 +601,108 @@ func TestMoveBalanceWithSignalError(t *testing.T) {
+				GasPriceModifier:       0.01,
+				MaxGasPriceSetGuardian: "100000",
+			},
+		},
+		EpochNotifier:       &epochNotifierMock.EpochNotifierStub{},
+		EnableEpochsHandler: enableEpochsHandlerMock.NewEnableEpochsHandlerStub(common.GasPriceModifierFlag),
+	})
+	require.Nil(t, err)
+
+	_ = economicsDataMock.SetTxTypeHandler(&testscommon.TxTypeHandlerMock{})
+
+	return economicsDataMock
+}
+
+func TestRelayedV3WithMultipleInnerTxs(t *testing.T) {
+	txHash := []byte("5c5cdb939473e76ee85af17eb02dbeca5d879bd19c7c050d0be6d2c901b9f6ae")
+
+	initialTx := &outportcore.TxInfo{Transaction: &transaction.Transaction{
+		Nonce:    47,
+		GasLimit: 500_000_000,
+		GasPrice: 1000000000,
+		SndAddr:  []byte("erd17ndrqg38lqf2zjgeqvle90rsn9ejrd9upx8evkyvh8e0m5xlph5scv9l6n"),
+		RcvAddr:  []byte("erd17ndrqg38lqf2zjgeqvle90rsn9ejrd9upx8evkyvh8e0m5xlph5scv9l6n"),
+		InnerTransactions: []*transaction.Transaction{
+			{
+				Nonce:    7,
+				SndAddr:  []byte("erd1e2ftj4hj43lkduwps9xdmtgjnmugkh9mndph4n2cxfmf6ufvn4ks0zut84"),
+				RcvAddr:  []byte("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"),
+				Data:     []byte("issueNonFungible@4d594e4654@414c4558@63616e467265657a65@74727565@63616e57697065@74727565@63616e5061757365@74727565@63616E5472616E736665724E4654437265617465526F6C65@74727565"),
+				GasPrice: 1000000000,
+				GasLimit: 60_000_000,
+			},
+			{
+				Nonce:    16,
+				SndAddr:  []byte("erd19yrjty2l4ytl6d3jynp5mqfekq4uf2x93akz60w7l3cp6qzny3psnfyerw"),
+				RcvAddr:  []byte("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"),
+				Data:     []byte("issueNonFungible@4d594e4654@414c4558@63616e467265657a65@74727565@63616e57697065@74727565@63616e5061757365@74727565@63616E5472616E736665724E4654437265617465526F6C65@74727565"),
+				GasPrice: 1000000000,
+				GasLimit: 60_000_000,
+			},
+			{
+				Nonce:    1,
+				SndAddr:  []byte("erd136rl878j09mev24gzpy70k2wfm3xmvj5ucwxffs9v5t5sk3kshtszz25z9"),
+				RcvAddr:  []byte("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"),
+				Data:     []byte("issueNonFungible@4d594e4654@414c4558@63616e467265657a65@74727565@63616e57697065@74727565@63616e5061757365@74727565@63616E5472616E736665724E4654437265617465526F6C65@74727565"),
+				GasPrice: 1000000000,
+				GasLimit: 60_000_000,
+			},
+		},
+	}, FeeInfo: &outportcore.FeeInfo{Fee: big.NewInt(0)}}
+
+	pool := &outportcore.TransactionPool{
+		Transactions: map[string]*outportcore.TxInfo{
+			hex.EncodeToString(txHash): initialTx,
+		},
+	}
+
+	arg := prepareMockArg()
+	arg.TxFeeCalculator = newEconomicsData(t)
+	txsFeeProc, err := NewTransactionsFeeProcessor(arg)
+	require.NotNil(t, txsFeeProc)
+	require.Nil(t, err)
+
+	err = txsFeeProc.PutFeeAndGasUsed(pool, 0)
+	require.Nil(t, err)
+	require.Equal(t, uint64(180_150_000), initialTx.GetFeeInfo().GetGasUsed())
+	require.Equal(t, "1074000000000000", initialTx.GetFeeInfo().GetFee().String())
+	require.Equal(t, "1074000000000000", initialTx.GetFeeInfo().GetInitialPaidFee().String())
```

Comment:
shouldn't these values be the same as in TestComputeAndAttachGasUsedAndFeeRelayedV3WithAllInnerTxFailed:
```go
uint64(160766000),
"2670920000000000"
"2864760000000000"
```